site/index.scroll

```diff
@@ -1,4 +1,7 @@
+title PLDB - A Comprehensive Database of Programming Languages and Features
+
 import settings.scroll
+htmlTitle PLDB - A Comprehensive Database of Programming Languages and Features
```

Comment:
What if we just kept it short:

```
htmlTitle PLDB - A Programming Language Database
```

Reply:
it's ok for me, the short version, potence the words in there, in this case, this 3 words Programming Language Database, and think those 3 words are the ones that best reflect what the PLDB is.